docs/pages/get-started/installation.md

```diff
@@ -4,50 +4,64 @@ title: Installation
 
 import { Terminal } from '~/ui/components/Snippet';
 
-There are two tools that you need to develop apps with Expo: a command line app called Expo CLI to initialize and serve your project and a mobile client app called Expo Go to open it on iOS and Android. Any web browser will work for opening the project on the web.
+To develop applications with Expo, you need two tools. A command-line application called [Expo CLI](#1-expo-cli) to initialize and serve your project, and a mobile client app called [Expo Go](#2-expo-go-app-for-ios-and) to open the project on iOS and Android platforms. You can use any web browser to run the project on the web.
```

Comment:
As we're no longer using `expo-cli` for initialization, I wonder if this should just say 'serve'?

Reply:
Great point. I forgot about the `create-expo-app` 😅 I have changed it.